### PR TITLE
Stats revamp v2 most popular time card

### LINKED ISSUE #16397: [android] Stats:  Most Popular Time Card
Existing card, needs update

### PR DESCRIPTION
**Pull request by ravishanker**
This PR updates the existing Most Popular Time card to new design. 

Fixes #16397 

| Before | New Design | Dark Theme |
|---|---|---|
|Screenshot_20220427_221059|Screenshot_20220427_220824|Screenshot_20220427_221413|

**Known issues**

- This currently doesn't cover the 'no data' case – this will come in a future PR:
- Styles will be updated in a separate PR

**To test**

Setup:

- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find StatsRevampV2FeatureConfig under Features in development enable it and restart app

Test 1:

- Launch app
- Go to stats either using quick links or menu
- Ensure that it opens in Insights tab otherwise switch to Insights tab
- Verify that the new Most Popular Time card is visible and matches the new designs above
- Ensure that 🔥 emoji is prefixed to the title, over flow menu is hidden and % views label below day and time
- If the Most Popular Time card is not present.  Go to stats management (either through ⚙️ menu at the top right hand corner or Add new stats card at the bottom) and add the card and save then return to Insights.

Test 2:

- Turn on the dark theme on your device
- Check that it card is displayed as in third image above

Test 3:

- Disable StatsRevampV2FeatureConfig and restart app
- Return to Stats and make sure Most Popular Time card is displayed as in first image above as before.

## Regression Notes
1. Potential unintended areas of impact
Existing Most Popular Time card will be update to new design.  No change in data

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/MostPopularInsightsUseCase.kt [hunk 1]
```diff
@@ -80,7 +80,8 @@ class MostPopularInsightsUseCase
         return items
     }
 
-    private fun buildTitle() = Title(R.string.stats_insights_popular, menuAction = this::onMenuClick)
+    private val fireEmojiTitle = "\uD83D\uDD25 ${resourceProvider.getString(R.string.stats_insights_popular)}"
```
Comment (irfano) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/MostPopularInsightsUseCase.kt]: Do you think we should add the emoji to the string resources, so translators can also manage the emoji?

File: WordPress/src/main/res/layout/stats_quick_scan_item.xml [hunk 2]
```diff
@@ -32,6 +32,15 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:text="@string/unknown" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/start_text"
+            style="@style/StatsBlockText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
```
Comment (irfano) [inline WordPress/src/main/res/layout/stats_quick_scan_item.xml]: np: Parent layout's height is height is `wrap_content`. So, I guess `android:layout_gravity="center_vertical"` has no fuction here. We can remove it. Also same for `end_text`.

**Review by irfano — CHANGES_REQUESTED** [review]
We can consider adding the feature flag. Because:
- We'll merge this to trunk. Known issues might be problems.
- All other old cards have three dots to open the menu on the top right side. If we don't use the feature flag, only this card won't have a three-dots menu button. It would be inconsistent, I think.

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/MostPopularInsightsUseCase.kt (hunk 1 above)
Comment (ravishanker) [reply WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/MostPopularInsightsUseCase.kt]: I couldn't get it to work in strings yesterday.  Found another way that works today

**Review by irfano — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**